dragonfly/model.py

```diff
@@ -0,0 +1,357 @@
+# coding: utf-8
+"""Dragonfly Model."""
+from ._base import _BaseGeometry
+from .properties import ModelProperties
+from .building import Building
+from .context import ContextShade
+
+from honeybee.shade import Shade
+from honeybee.boundarycondition import Surface
+from honeybee.facetype import AirWall
+from honeybee.typing import float_in_range
+
+from ladybug_geometry.geometry2d.pointvector import Vector2D
+
+import math
+
+
+class Model(_BaseGeometry):
+    """A collection of Buildings and ContextShades for an entire model.
+
+    Properties:
+        * name
+        * display_name
+        * north_angle
+        * north_vector
+        * buildings
+        * context_shades
```

Comment:
Shouldn't a model also have a min and max property?

Reply:
Sure. I don't know how I would use it right now but it's an easy thing to add so I'll just put it in.